.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,9 +21,9 @@ Please ensure that the following checklist has been satisfied before submitting
 - [ ] All code follows the
       [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
       [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
+      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
       and
       [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
       style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
 - All code has been tested:
   - [ ] Unit tests have been written where possible.
-  - [ ] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.
```

Comment:
I don't think routine testing of the Electron package is necessary now--for any PRs that don't specifically work on the build process, the usual `npm run dev` can be expected to be a valid way to test GUI functionality.

Reply:
I think this point was more about testing the IDE (with actual Backend) than about the exact way it was done. 
I think it is good to go as it was largely dead rule, but it might be good to confirm this with the management.